### PR TITLE
Translating 2019 Fonts chapter to Spanish

### PR DESCRIPTION
**Pull request by c-torres**
Progress on #498

### CODE REVIEW

File: src/content/es/2019/fonts.md [hunk 1]
```diff
@@ -0,0 +1,651 @@
+---
+part_number: I
+chapter_number: 6
+title: Fonts
```
Comment (bazzadp) [inline src/content/es/2019/fonts.md:4]: Should this be Fuentes?

```suggestion
title: Fuentes
```

File: src/content/es/2019/fonts.md [hunk 2]
```diff
@@ -0,0 +1,651 @@
...
+</figure>
+
+Incluso con un 1.8%, esto fue más alto de lo esperado, aunque estoy emocionado de ver esto despegar. [Google Fonts v2](https://developers.google.com/fonts/docs/css2) incluye cierto soporte para fuentes variables.
+
+<figure>
+  <a href="/static/images/2019/fonts/fig19.png">
+    <img src="/static/images/2019/fonts/fig19.png" alt="Figura 19. Uso de los ejes 'font-variación-settings'." aria-labelledby="fig19-caption" aria-describedby="fig19-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vQDogXDb3BwZZHrBT39qccP_LJoCScD3QEi_FmjT_8VDPD_1Srpz-g7ZuuTUEb8pYXBpDmQzZ1hQh7q/pubchart?oid=699343351&amp;format=interactive">
+  </a>
+  <div id="fig19-description" class="visually-hidden">Bar chart showing the usage of the font-variation-settings property. 42% of properties on desktop pages are set to the "opsz" value, 32% to "wght", 16% to "wdth", 2% or fewer to "roun", "crsb", "slnt", "inln", and more. The most notable differences between desktop and mobile pages are 26% usage of "opsz", 38% of "wght", and 23% of "wdth".</div>
```
Comment (bazzadp) [inline src/content/es/2019/fonts.md:628]: Missed the translation here

**Review by bazzadp — CHANGES_REQUESTED** [review]
Looks pretty good to me from a technical side!

A few questions/corrections.

Also would it be possible for you to translate Zach's Bio at same time as it appears at the bottom of this page?

https://github.com/HTTPArchive/almanac.httparchive.org/blob/3f06faaa5792fcac9881ae6415ff855e7f1021fa/src/templates/es/2019/base.html#L202

And on a similar note can we also translate the Featured Quote from this chapter:

https://github.com/HTTPArchive/almanac.httparchive.org/blob/3f06faaa5792fcac9881ae6415ff855e7f1021fa/src/templates/es/2019/featured_chapters.html#L22-L24

And also enable this Featured Chapter by changing line 1 of that template:

https://github.com/HTTPArchive/almanac.httparchive.org/blob/3f06faaa5792fcac9881ae6415ff855e7f1021fa/src/templates/es/2019/featured_chapters.html#L1

to:
```
{%- set featured_chapter = ("third-parties","fonts") | random %}
```

**Comment by c-torres** [comment]
@bazzadp thanks for the feedback, I believe all the suggestions have been applied.

**Comment by bazzadp** [comment]
Wow that was quick! Thanks!

We really should translate the bios and featured chapters of the other translated chapters at some point. But we'll leave that for another day and another PR.

**Review by bazzadp — APPROVED** [review]

**Comment by bazzadp** [comment]
Planning a release this evening so gonna merge this as is. @garcaplay is you get a chance to review at a later date then feel free to comment here, or open a new PR.

**Comment by garcaplay** [comment]
Hi guys! Sorry for the late response! As soon as I have the chance will review this new code (with a more language oriented view, as I'm pretty sure you covered a 100% of the technical perspective on your review ;) ) and if find something else to fix will let you know!

**Comment by bazzadp** [comment]
No problems!

Just to let you know when I’m doing a release I often will merge unreviewed translations we have queued up at the time. While of course it’s best to get a translation reviewed by another where possible, I do appreciate that people are doing this on their own spare time so have to squeeze it in where they can. In the meantime I figure a translation by a native speaker is better than no translation at all, even if it may contain some typos or things that could be phrased better. We can always tweak it later. So I would rather include in the release than not. At the same time don’t want to pressure reviewers or demand a review by a set time, so figure this is the best way.

Hopefully that explains my thinking and do let me know if you’ve any suggestions on better ways or working?

Thanks again (to both of you!) for your efforts on this!

**Pull request merged**